ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/StateTreeManager.java

```diff
@@ -59,8 +59,11 @@ public void onChainStarted(ChainStartEvent event) {
     // TODO: Startup Logic: Initial State Data
     List<Bytes32> merkle_roots = new ArrayList<Bytes32>();
     merkle_roots.add(Bytes32.ZERO);
+    List<Bytes48> proof_of_posession = new ArrayList<Bytes48>();
+    proof_of_posession.add(Bytes48.random());
+    proof_of_posession.add(Bytes48.random());
     DepositInput deposit_input =
-        new DepositInput(Bytes48.ZERO, Bytes32.ZERO, new ArrayList<Bytes48>());
+        new DepositInput(Bytes48.random(), Bytes32.random(), proof_of_posession);
```

Comment:
There's a `randomDepositInput()` static method in _DataStructureUtil.java_. Might be good to use that for maintainability (though, I guess this is temporary pending real data).

Reply:
Love this.  Thank you